aikau/src/main/resources/alfresco/services/PreviewService.js

```diff
@@ -50,18 +50,20 @@ define(["dojo/_base/declare",
        * @instance
        */
       generateDependencies: function alfresco_services_PreviewService__generateDependencies(payload) {
+         /*jshint eqnull:true*/
          if (payload != null && payload.model != null)
          {
-            var pageDefObject = dojoJson.parse(payload.model);
-            var data = {
-               jsonContent: pageDefObject,
-               widgets: payload.model
-            };
+            // This block (and query below) commented out because JSHint said they were unused .... why unused though? Why are they here?
```

Comment:
Code probably got copied somewhere else during a refactor. Happy to leave this comment in the code for now and investigate at a later date